README.md

```diff
@@ -14,7 +14,7 @@ Vita3K is an experimental PlayStation Vita emulator for Windows, Linux and macOS
 The emulator currently runs some homebrew programs. It is also able to load some decrypted commercial games.
 
 - [Homebrew compatibility database](https://github.com/Vita3K/homebrew-compatibility/issues)
-- [Commerical compatiblity database](https://github.com/Vita3K/compatibility/issues)
+- [Commercial compatibility database](https://github.com/Vita3K/compatibility/issues)
```

Comment:
put this on a separate commit, has nothing to do with the main one

Reply:
Done

Reply:
it's still on 1 commit

Reply:
You can keep the change just make another commit for it. :)

Reply:
I tried the force push thing, to be honest, I hate rebases and all that, super un-intuitive on top of already un-intuitive git itself.
What you guys are saying is that this README change is still here somewhere?

Reply:
what @1whatleytay said

Reply:
No, we're saying it's gone and we like it :D 

We would rather it be in a second commit though.

Reply:
Gotcha for some reason I read a separate PR. Done.

Reply:
nah we're not that strict :p